rotkehlchen/tests/api/test_custom_assets.py

```diff
@@ -24,6 +24,7 @@
 from rotkehlchen.tests.utils.api import (
     api_url_for,
     assert_error_response,
+    assert_proper_response,
```

Comment:
generic comment for this file. This is testing manually input tokens, and not custom assets created with the new custom_assets table, right? So can you perhaps rename this file and all the functions to make it obvious?

Perhaps `custom_assets` table also needs a better name but nothing comes to mind.

Reply:
Renamed this file and methods to `user assets`. Used this name since there is an endpoint for this purpose with the this name https://github.com/rotki/rotki/blob/develop/rotkehlchen/api/server.py#L277